lib/auth/tun_test.go

```diff
@@ -138,7 +138,9 @@ func (s *TunSuite) TestUnixServerClient(c *C) {
 	otpSecret := base32.StdEncoding.EncodeToString([]byte(rawSecret))
 
 	user, role := createUserAndRole(s.a, userName, []string{userName})
-	role.SetResource(services.KindNode, services.RW())
+	rules := role.GetRules(services.Allow)
```

Comment:
@klizhentas earlier we have agreed to simplify `Role` interface and only allow setting all rules at once. This keeps the interface small and allows better validation. The shortcut like this would only make sense in a test environment. Nowhere in Teleport "resource rules" are even needed.